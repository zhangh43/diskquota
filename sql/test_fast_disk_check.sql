-- Test SCHEMA
CREATE SCHEMA s1;
SET search_path to s1;

CREATE TABLE a(i int);
INSERT INTO a SELECT generate_series(1,2000000);
SELECT pg_sleep(20);
SELECT (pg_database_size(oid)-dbsize)/dbsize < 0.1  FROM pg_database, diskquota.database_size_view WHERE datname='contrib_regression';
RESET search_path;
DROP TABLE s1.a;
DROP SCHEMA s1;

