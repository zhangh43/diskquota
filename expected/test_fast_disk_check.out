-- Test SCHEMA
CREATE SCHEMA s1;
SET search_path to s1;
CREATE TABLE a(i int);
NOTICE:  Table doesn't have 'DISTRIBUTED BY' clause -- Using column named 'i' as the Greenplum Database data distribution key for this table.
HINT:  The 'DISTRIBUTED BY' clause determines the distribution of data. Make sure column(s) chosen are the optimal data distribution key to minimize skew.
INSERT INTO a SELECT generate_series(1,2000000);
SELECT pg_sleep(20);
 pg_sleep 
----------
 
(1 row)

SELECT (pg_database_size(oid)-dbsize)/dbsize < 0.1  FROM pg_database, diskquota.database_size_view WHERE datname='contrib_regression';
 ?column? 
----------
 t
(1 row)

RESET search_path;
DROP TABLE s1.a;
DROP SCHEMA s1;
